thermal/Thermal.cpp

```diff
@@ -258,14 +263,23 @@ static int thermal_get_cpu_usages(CpuUsage *list)
             online = 1;
         }
 
+	//Current Max size supported is 16 cores, for more CPU add check
+	int length = sizeof(CPU_LABEL) / sizeof(CPU_LABEL[0]);
```

Comment:
Declare length variable in the starting of function where other variables are declared.